tests_integration/test_temporary_connection.py

```diff
@@ -13,30 +13,39 @@
 # limitations under the License.
 
 import pytest
+import os
+from unittest import mock
 
 
 @pytest.mark.integration
-def test_temporary_connection(runner):
+@mock.patch.dict(
+    os.environ,
+    {
+        "SNOWFLAKE_CONNECTIONS_INTEGRATION_ACCOUNT": os.environ.get(
+            "SNOWFLAKE_CONNECTIONS_INTEGRATION_ACCOUNT", None
+        ),
+        "SNOWFLAKE_CONNECTIONS_INTEGRATION_USER": os.environ.get(
+            "SNOWFLAKE_CONNECTIONS_INTEGRATION_USER", None
+        ),
+        "SNOWFLAKE_PASSWORD": os.environ.get(
+            "SNOWFLAKE_CONNECTIONS_INTEGRATION_PASSWORD"
+        ),
+    },
+    clear=True,
+)
+def test_temporary_connection(runner, snapshot):
```

Comment:
Right now there is only one performance test.
Do we plan to add next in the future?
Maybe it would be better to add new fixture `performance_runner` instead of mocking ?
WDYT?